### ISSUE TITLE
4.0: UI redirects to 404 page

### ISSUE THREAD

**Issue by nilshoerrmann** [distantnative/retour-for-kirby#282]
We have paths like `projekte-mit-filter/tag/Kultur%252520%252526%252520Denkmalschutz.html` coming from a legacy CMS. Clicking these entries in the table of routes results in a redirect to `/panel/dialogs/retour/redirects/projekte-mit-filter%2Ftag%2FKultur%252520%252526%252520Denkmalschutz.html/edit?column=from` which returns a 404 page.

Other routes work just fine. 
This is the corresponding definition:

```yaml
  - 
    from: 'projekte-mit-filter/tag/Kultur%2520%2526%2520Denkmalschutz.html'
    to: projekte/kategorie:kultur-denkmalschutz
    status: 301
    priority: false
    comment: null
```

**Comment by nilshoerrmann** [comment]
Removing all slashes from the `from` key fixes the issue – this breaks the given URL of course.

**Comment by distantnative** [comment]
Can't reproduce this. You refer to clicking the little chain icon in the Panel view in the "Path" column?

**Comment by nilshoerrmann** [comment]
No, I was clicking the text.

**Comment by distantnative** [comment]
Clicking the text in the table? That should just open the dialog to edit and focus the field - which it does for me, cannot reproduce any redirecting for me.

What browser are you using on which OS?

**Comment by nilshoerrmann** [comment]
Opening the dialog works for all links but the mentioned ones.

This is happening with latest Firefox on macOS Monterey, localhost via MAMP.

**Comment by distantnative** [comment]
Flawless for me with the latest build (also just released as 4.0.0). Downloaded Firefox, all fine for me.


https://user-images.githubusercontent.com/3788865/155856402-a2fb54d0-6dfe-4728-b932-3e555bd8eff8.mov



If it still remains for you with that - maybe what Kirby version are you using?

**Comment by nilshoerrmann** [comment]
I just checked this again, updating Kirby and Retour to their latest version, and this is still happening on localhost as well as on the live server. This is not limited to Firefox and also happens using Safari for instance.

By the way, the actual redirect on the frontend works as expected, it's only the call to the panel dialog that is failing.

I have no idea what's causing this and as you cannot reproduce the issue, let's just close it. I'll reopen the issue in case this happens again and I can provide a better case for reproduction.
